docs/modules/quick-start/pages/situations.adoc

```diff
@@ -0,0 +1,51 @@
+
+:imagesdir: ../assets/images
+= Step 2: View and Create Situations
+
+Now that you have xref:engine-config.adoc[selected the Clustering engine], ALEC automatically begins to identify situations.
+This requires new alarms to be triggered within your OpenNMS environment, so--depending on your network's usual activity--this may take some time.
+
+== View situations
+
+After alarms have been triggered in OpenNMS, they are evaluated by the Clustering engine and grouped into clusters as appropriate.
```

Comment:
```suggestion
After alarms have been triggered in OpenNMS, the clustering engine evaluates and groups them into situations as appropriate.
```
As per earlier question. Is there a distinction between a cluster and a situation?